src/main/java/io/swagger/codegen/v3/generators/java/AbstractJavaCodegen.java

```diff
@@ -167,6 +169,7 @@ public AbstractJavaCodegen() {
         java8ModeOptions.put("false", "Various third party libraries as needed");
         java8Mode.setEnum(java8ModeOptions);
         cliOptions.add(java8Mode);
+        cliOptions.add(CliOption.newBoolean(NOT_NULL_JACKSON_ANNOTATION, "Option: to set Jackson to ignore null fields when serializing"));
```

Comment:
use the following for description: "adds @JsonInclude(JsonInclude.Include.NON_NULL) annotation to model classes"

Reply:
done